Controller/Api/BlockController.php

```diff
@@ -26,6 +24,8 @@
  */
 class BlockController
 {
+    use FOSRestApiTrait;
```

Comment:
We support php 5.3, so we don't have access to traits, sorry… use inheritance instead.